ravenframework/Optimizers/crossOverOperators/crossovers.py

```diff
@@ -103,7 +103,7 @@ def uniformCrossover(parents,**kwargs):
     children1,children2 = uniformCrossoverMethod(parent1,parent2,crossoverProb)
     children[index]   = children1
     children[index+1] = children2
-    index = index + 1
+    index +=  2
```

Comment:
why index+= 2 instead of index+= 1?

Reply:
![image](https://user-images.githubusercontent.com/110840088/195879090-015a44c7-a296-40dc-a769-10646c795f9c.png)

if index += 1, then one of the children values are overwritten as iteration goes.

Reply:
The figure below is the result of uniformCrossover unit test if index += 1:
![image](https://user-images.githubusercontent.com/110840088/195880129-7352d088-e729-4cdf-a437-78eedcd67c75.png)

Reply:
The figure below is the result if index += 2:
![image](https://user-images.githubusercontent.com/110840088/195880417-484bccbe-b145-41e9-9f95-f608cb2bb8ec.png)